security/security-core/src/main/java/io/camunda/security/configuration/BasicAuthenticationConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.security.configuration;
+
+public class BasicAuthenticationConfiguration {
+  private boolean apiDangerouslyUnprotected = false;
```

Comment:
🔧  The name made my day 😄  still as this is operator/admin facing I would be in for a simpler less fancy name. Wdyt of `camunda.security.authentication.basic.api.allow-unauthenticated=false`

Reply:
Just aligned with PM, we would prefer to not replicate the react approach/naming and go with `camunda.security.authentication.basic.api.allow-unauthenticated=false` or something similar without warnings in the config name. We can use those words in the docs.

I get the intention but as the default will be false + the app logs a warning and the config name cleary indicates that unauthenticated requests are allowed I see no reason strong for including `dangerous` in the config name.